npm-shrinkwrap.json

```diff
@@ -340,7 +340,7 @@
     "fxa-auth-db-mysql": {
       "version": "1.105.0",
       "from": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#master",
-      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#71445b16bcb8eb00185908331fececf84b55eb7f",
+      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#6a4fb6771d100bfb0fad4b1939df2ea1d9bcfe2f",
```

Comment:
Can you do a full `npm run shrink` while you're updating this? I want shrinkwrap to pick up the `0.0.7` speed-trap, and that would save me the effort! 😄

Reply:
Sorry @vbudhram, there are no words to describe how stupid this ask was. As you pointed out in IRC, wrong repo. 😆

Reply:
No worriees, did a full npm shrink anyways 👍